src/main/java/nextstep/courses/domain/Sessions.java

```diff
@@ -0,0 +1,46 @@
+package nextstep.courses.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class Sessions {
+    private static final String ALREADY_SESSION = "이미 등록된 강의 입니다.";
+    private static final String NOT_INFO = "등록된 정보가 없습니다.";
+    private static final String NULL_EXCEPTION = "강의가 잘못 생성되었습니다.";
+    private Map<Long, Session> sessionMap = new HashMap<>();
```

Comment:
```suggestion
    private final Map<Long, Session> sessionMap = new HashMap<>();
```

인스턴스 변수는 가능하면 final로 구현을 시작하고 변경이 필요한 시점에 final을 제거하는 방식으로 구현하면 어떨까?